projects/lendflare/utils.js

```diff
@@ -0,0 +1,76 @@
+const sdk = require('@defillama/sdk');
+// const { getTokenPrices } = require('@defillama/sdk/build/computeTVL/prices');
+const { toUSDTBalances } = require('../helper/balances')
+const utils = require("../helper/utils");
+
+const COINS_LIST = `dai,bitcoin,ethereum,wrapped-bitcoin,frax,usd-coin,tether,fei-usd,true-usd,maker`;
+
+const poolLength = {
+    "inputs": [],
+    "name": "poolLength",
+    "outputs": [
+        {
+            "internalType": "uint256",
+            "name": "",
+            "type": "uint256"
+        }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+}
+
+// ethereum
```

Comment:
repeat of the comment, the block needs to be passed to the methods in this file as well